profile/README.md

```diff
@@ -10,3 +10,33 @@
 🍿 Fun facts - what does your team eat for breakfast?
 🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
 -->
+
```

Comment:
It would be nice to have a short intro as you did here https://github.com/uclahs-cds#welcome-to-the-team- and link to the official UCLA lab link https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab

Reply:
I moved some of the words into the intro from that link and also added the link separately under "more info" just below the intro.